pandas/core/generic.py

```diff
@@ -2762,6 +2762,7 @@ def to_latex(
         index=True,
         na_rep="NaN",
         formatters=None,
+        formatters_col=None,
```

Comment:
To not break API compatibility for user who are calling this function positionally, this kwarg should go at the end.